extensions/binding/org.eclipse.smarthome.binding.dmx/ESH-INF/thing/artnet-bridge.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dmx"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- ArtNet Bridge -->
+    <bridge-type id="artnet-bridge">
+        <label>ArtNet Bridge</label>
+        <description>The ArtNet bridge represents a single DMX universe connected via ArtNet, only unicast</description>
+
+        <channels>
+            <channel id="mute" typeId="mute"/>        
+        </channels>
+
+        <config-description>
+            <parameter name="address" type="text">
+                <context>network_address</context>
+                <label>Network Address</label>
+                <description>Network address of bridge, format: address[:port]. Default port is 6454.</description>
+                <required>true</required>
+            </parameter>
+            <parameter name="localaddress" type="text">
+                <context>network_address</context>
```

Comment:
I thought about that and decided against. The reason is: Most "address"-fields allow lists, where you can use different receivers. This is in the form "address[:port],address[:port]". If address and port are splitted into different fields, it would be mandatory (and possibly easy to get wrong) to add ports for every address. Example:

192.168.0.0,192.168.1.3:4000,192.168.4.2:5000, 10.0.1.3

The first and last would use the default port, while the second and third the port specified. If you split it, you would need to specify the default port, too, so

192.168.0.0,192.168.1.3,192.168.4.2,10.0.1.3
6454,4000,5000,6454

This is hard to understand. To keep address specification constistent, I used the same format for the local address.